handlers.py

```diff
@@ -48,6 +49,34 @@ async def reply(self, content):
         self.write(content)
         await self.finish()
 
+    def prepare(self):
+        ip_address = self.request.headers.get("X-Real-IP") or \
+                     self.request.headers.get("X-Forwarded-For") or \
+                     self.request.remote_ip
+        print(ip_address)
+
+        #if self.request_is_limited(key=ip_address, limit=self.application.limit, period=self.application.period):
```

Comment:
commented this out, cause I want to first make sure we recognize the ip correctly

Reply:
I'm not sure we can see prints on heroku - use logging